### PR TITLE
unify syntax in readmes

### PR DESCRIPTION
**Pull request by petrparolek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,17 +74,16 @@ composer create-project -s dev planette/nutella-project acme
...
-    parameters:
-
-        # Database
-        database:
-            host: localhost
-            dbname: nutella
-            user: nutella
-            password: nutella
-    ```
+```neon
```
Comment (f3l1x) [inline README.md:77]: Odsazeni

File: README.md [hunk 2]
```diff
@@ -115,17 +114,16 @@ composer create-project -s dev planette/nutella-project acme
...
-    parameters:
-
-        # Database
-        database:
-            host: database
-            dbname: nutella
-            user: nutella
-            password: nutella
-    ```
+```neon
```
Comment (f3l1x) [inline README.md:117]: Odsazeni

**Review by f3l1x — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (petrparolek) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (petrparolek) [reply README.md]: ok

**Pull request merged**